packages/react-tabs/src/components/Tab/Tab.tsx

```diff
@@ -12,6 +13,7 @@ export const Tab: ForwardRefComponent<TabProps> = React.forwardRef((props, ref)
   const state = useTab_unstable(props, ref);
 
   useTabStyles_unstable(state);
+  useTabAnimatedIndicatorStyles_unstable(state);
```

Comment:
What is the motivation behind calling this hook from the top level `Tab` component? This is against our API, since now there is another hook that users must know about if they are thinking of recomposing the component. We have made an exception for `useXXXContext` before since that could be applied for any component that would leverage react context.

In this case I don't think `useXXXAnimatedIndicatorStyles` would be used in many places and right now seems tab specific.

Reply:
Moved and not exported.